src/panels/TimePanel/StepControls.tsx

```diff
@@ -27,19 +28,19 @@ export function HoldableButton({
   const shouldUseStepInterval =
     stepHoldDelay !== undefined && stepHoldInterval !== undefined;
 
-  function downHandler(event: KeyboardEvent) {
+  function downHandler(event: KeyboardEvent): void {
```

Comment:
```suggestion
  function keyDownHandler(event: KeyboardEvent): void {
```
otherwise, it might just as well be a mousedown handler